docs/topics/install.md

```diff
@@ -87,8 +87,100 @@ Brigade creates Persistent Volume Claims on the fly. By default it will create
 PVCs using the default persistent volume class. Changing your cluster's default
 storage class will change what Brigade creates.
 
-We recommend the following file system types for these distributions or platforms:
+The following volume backends have been tested:
 
 - Azure: AzureFile
 - Minikube: 9p (the default)
-- General: A storage type that supports ReadWriteMany.
+- Any: NFS
+
+The NFS solution is explained below.
+
+#### Using an NFS Server
+
+Brigade uses storage for caching and short-term file sharing. Because of this,
+it is often convenient to use storage backends that are optimized for short-term
+ephemeral storage.
+
+NFS (Network File System) is one protocol that works well for Brigade. You can
+use the [NFS Provisioner](https://github.com/IlyaSemenov/nfs-provisioner-chart)
+chart to easily install an NFS server.
+
+```console
+$ helm repo add nfs-provisioner https://raw.githubusercontent.com/IlyaSemenov/nfs-provisioner-chart/master/repo
+$ helm install --name nfs nfs-provisioner/nfs-provisioner --set hostPath=/var/run/nfs-provisioner
+```
+
+(Note that RBAC is enabled by default. To turn it off, use `--set rbac.enabled=false`.)
+
+To use an emptyDir instead of a host mount, set `--hostPath=""`.
+
+
+```console
+$ helm repo add nfs-provisioner https://raw.githubusercontent.com/IlyaSemenov/nfs-provisioner-chart/master/repo
+$ helm install --name nfs nfs-provisioner/nfs-provisioner --set hostPath=""
+```
+
+If you have plenty of memory to spare, and are more concerned with fast storage,
+you can configure the provisioner to use a tmpfs in-memory filesystem.
+
+```console
+$ helm install --name nfs nfs-provisioner/nfs-provisioner --set hostPath="" --set useTmpfs=true
+```
+
+This chart installs a `StorageClass` named `local-nfs`. Brigade projects can
+each declare which storage classes they want to use. And there are two storage
+class settings:
```

Comment:
In lieu of a named StorageClass, this can be installed as a default storage class. Though an explicit name is recommended.
`$ helm install --name nfs --namespace nfs-provisioner nfs-provisioner/nfs-provisioner --set defaultClass=true`

Reply:
That's a good idea.